src/server/controllers/rating.js

```diff
@@ -28,7 +31,26 @@ const postRating = async (req, res) => {
     res.json({ data: createdRating })
 }
 
+const seedRating = async (req, res) => {
+
+    for (let i = 1; i <= pokeNumber; i++) {
+        const pokemonId = i;
+        const seededRating = await prisma.rating.create({
+            data: {
+                profileId: 1,
+                pokemonId: pokemonId,
+                strength: getRandom(1, 6),
+                companion: getRandom(1, 6),
+                design: getRandom(1, 6)
```

Comment:
why 6?

Reply:
because I'm using Math.floor() in the function to return an int, which means the max value isnt inclusive `(1 <= x < 6)`

Reply:
I'm happy with it. Could also have min and max variables that you call?

Reply:
could you make some constants that make it clearer what those values are?